.github/workflows/python-app.yml

```
@@ -13,24 +13,72 @@ permissions:
-  build:
-
+  backend-tests:
+    name: Backend Tests
     runs-on: ubuntu-latest
-
+    
     steps:
     - uses: actions/checkout@v4
+    
     - name: Set up Python 3.13
       uses: actions/setup-python@v3
```

actionlint (1.7.7)

the runner of "actions/setup-python@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)